bundle-workflow/Pipfile

```diff
@@ -9,6 +9,9 @@ requests = "~=2.26"
 isort = "~=5.9"
 flake8 = "~=3.9"
 pytest = "*"
+boto3 = "1.18.31"
+botocore = "1.21.33"
+json2table = "1.1.5"
```

Comment:
There are a couple of libraries that do this [json2html](https://pypi.org/project/json2html), is there a reason you picked this one over others?

Reply:
json2table gives more options in terms of rotating the table from vertical to a hroizontal view as well as table attributes.